src/main/java/com/enderio/base/common/item/tool/YetaWrenchItem.java

```diff
@@ -47,7 +43,7 @@ public InteractionResult onItemUseFirst(ItemStack stack, UseOnContext pContext)
                 // Cycle state.
                 optSideConfig.ifPresent(ISideConfig::cycleMode);
                 return InteractionResult.SUCCESS;
-            }
+            }else return InteractionResult.PASS;
```

Comment:
On a return, we don't generally have to put an else, but its stylistic choice.
Also check your spacing :)